tasks/DroneFlightControlTask.cpp

```diff
@@ -237,14 +256,15 @@ void DroneFlightControlTask::goTo()
     base::Vector3d position = getRigidBodyState().position;
     // Convert to NEU
     position[1] = -position[1];
+    mCmdPos.position[1] = -mCmdPos.position[1];
+    // get offset
     base::Vector3d offset = (mCmdPos.position - position);
     // Convert to deg!
     float yawDesiredInDeg = (mCmdPos.heading.rad) * 180 / M_PI;
 
     float32_t xCmd = static_cast<float>(offset[0]);
     float32_t yCmd = static_cast<float>(offset[1]);
-    float32_t zCmd = static_cast<float>(mCmdPos.position[2]) +
-                                        static_cast<float>(getRigidBodyState().position[2]);
+    float32_t zCmd = static_cast<float>(mCmdPos.position[2]);
```

Comment:
Do the NWU-to-NED conversion here.

Reply:
The cmd is NEU. It is just necessary to change y signal, in that case.